lib/masamune/actions/data_flow.rb

```diff
@@ -58,7 +65,7 @@ def parse_file_type(key)
       Set.new File.read(value).split(/\s+/)
     end
 
-    def execute(options = {})
+    def prepare_and_execute(options = {})
```

Comment:
- mark this function as `private` since it doesn't need to be called by consumers
- rename to prevent function aliasing when including `Masamune::Actions::Execute`